### ISSUE TITLE
mqtt Disconnect when ssl enabled with STA+AP Mode

### ISSUE THREAD

**Issue by jassi00713** [marvinroger/async-mqtt-client#106]
hey

i am using your library to make ssl mqtt connections to a local mqtt broker.
my setup is as follows:
1) BROKER   : a local mosquito broker
2) GATEWAY : an esp8266 in STA+AP Mode
3) NODE        : an esp8266 in STA Mode which connects to the GATEWAY

the gateway makes a successful connection to the local broker over ssl.
Everything looks fine, i am able to publish and subscribe to topics successfully and handle messages.

now, when the NODE connects to the GATEWAY, all is still fine. Normal, TCP Communication takes place between them(i am using ESPAsyncTCP library for that). But when i do an mqtt publish or mqtt subscribe, the mqtt disconnects with reason 0.

In short, the mqtt publish or mqtt subscribe commands work fine when GATEWAY is connected to local broker over wifi with GATEWAY's AP on.

But when a NODE connects to that GATEWAY, the mqtt publish or subscribe commands lead to mqtt disconnect with reason 0 i.e. TCP lost.

Kindly, help me in this issue

**Comment by bertmelis** [comment]
Your node also has to connect to the broker, via the gateway.
